Workbooks/Windows Virtual Desktop/NoData/NoData.workbook

```diff
@@ -362,30 +362,12 @@
                 }
               ]
             },
-            "name": "ConfigWorkbook"
-          },
-          {
-            "type": 11,
-            "content": {
-              "version": "LinkItem/1.0",
-              "style": "list",
-              "links": [
-                {
-                  "id": "beb23a72-0459-426f-8ee2-3e5c8a31cfc9",
-                  "cellValue": "{HostPool}",
-                  "linkTarget": "Resource",
-                  "linkLabel": "Go to AVD Insights legacy",
-                  "subTarget": "insightsLegacy",
-                  "style": "link"
-                }
-              ]
-            },
             "conditionalVisibility": {
               "parameterName": "UsingAMA",
               "comparison": "isEqualTo",
               "value": "yes"
             },
-            "name": "LegacyLink"
+            "name": "ConfigWorkbook"
```

Comment:
nit: more precise naming here might help avoid name conflicts and increase clarity. For example, line 245 already uses the name `configWorkbook`.

This element is not the Config Workbook, but it might be the `ConfigWorkbookReference` or something similar.